bodhi/tests/server/services/test_releases.py

```diff
@@ -279,6 +289,52 @@ def test_edit_release(self):
         r = self.db.query(Release).filter(Release.name == name).one()
         self.assertEquals(r.state, ReleaseState.current)
 
+    def test_edit_release_critpath_pkgs(self):
+        """
+        Assert that editing critical path packages for a release are saved correctly.
+        """
+        name = u"F22"
+
+        res = self.app.get('/releases/%s' % name, status=200)
+        r = res.json_body
+
+        # Test adding a package.
+        r["edited"] = name
+        r["csrf_token"] = self.get_csrf_token()
+        r["critpath_pkgs"] = "bodhi"
```

Comment:
We don't want there to be two ways to change this field - let's drop this method in favor of the PUT/DELETE calls.